__docs__/wonder-blocks-banner/banner.stories.tsx

```diff
@@ -468,6 +470,68 @@ export const WithMixedActions: StoryComponentType = {
     ),
 };
 
+/**
+ * Use the `icon` prop to show a specific Phosphor icon in the banner instead. If the
+ * `icon` prop is not set, a default icon will be used in the banner depending
+ * on the `kind` prop.
+ *
+ * __NOTE:__ Icons are available from the [Phosphor
+ * Icons](https://phosphoricons.com/) library.
+ *
+ * To use a Phosphor icon, you can use the following syntax:
+ *
+ * ```jsx
+ * import magnifyingGlass from "@phosphor-icons/core/regular/magnifying-glass.svg";
+ * <Banner icon={magnifyingGlass} layout="floating" text="text" />
+ * ```
+ *
+ * __Accessibility__: The icon chosen for the banner is decorative and
```

Comment:
Let me know what your thoughts are on letting the `kind` prop be the `aria-label` for the icon! (Semi-related: I created WB-1713 for supporting translations for this aria-label)

An alternative option I was considering was adding a new prop for the aria-label of the icon so that devs can provide their own aria-label. After seeing the examples of icons that could be used in the banner (like the clock icon or Khanmigo icon), these seem to be more decorative and the `kind` label is more helpful to communicate the status

Reply:
hmmm good point. I think mapping by default to the `kind` prop makes sense, but I also see your point about using a separate `aria-label` prop.

So yeah, I like of your idea about adding a new `aria-label` for the icon (I understand it would still default to `kind`, right?).

Reply:
I'm wondering if the `kind` should always be there and communicated to screen readers since we only use colour to communicate that the message is critical/warning/info/success. Or would the guideline be that if teams provide their own `aria-label` for the icon, it should include details about the banner `kind`?

Reply:
I think they should rely on the default `aria-label` but they could optionally override it if they want (or need to). That means that we could create an `iconAriaLabel` (or such) optional prop in a separate ticket as a lower priority. wdyt?

Reply:
That sounds good to me! I created WB-1715 for this, thanks for the feedback!